app/helpers/mission_type_helper.rb

```diff
@@ -0,0 +1,20 @@
+module MissionTypeHelper
+  def self.border_style(mission_type)
```

Comment:
I think it should be possible to write this shorter, something like...

```Ruby
def self.border_style(mission_type)
  valid_numbers = [1..6]
  return "mission-1" unless valid_numbers.include? mission_type

  "mission_#{mission_type}"
end
```

Reply:
I'm separating these into a new PR to make the helper available for other components requiring mission specific styles https://github.com/alphagov/frontend/pull/4416.